### PR TITLE
Store LRU cache of last filled ratelimit buckets

### PR DESCRIPTION
**Pull request by left-try**
I have added  LRU cache as a `ZSET` of overlimited bucket's prefixes with timestamps as scores of fixed size 30.

Issue: #5147

### CODE REVIEW

File: src/plugins/lua/ratelimit.lua [hunk 1]
```diff
@@ -441,7 +443,7 @@ local function ratelimit_cb(task)
 
     for pr, value in pairs(prefixes) do
       local bucket = value.bucket
-      local rate = (bucket.rate) / 1000.0 -- Leak rate in messages/ms
+      local rate = (bucket.rate) / 1000.0 -- Leak rate in messast filled bucketsages/ms
```
Comment (vstakhov) [inline src/plugins/lua/ratelimit.lua:446]: wut?

Comment (left-try) [reply src/plugins/lua/ratelimit.lua:446]: idk

**Comment by vstakhov** [comment]
Another hint: you don't need to use `draft` in your PR name - just make it draft!

**Comment by left-try** [comment]
Also a pull request for issue #5147 
It is about creating LRU cache for last buckets that are over limit.
Sorry for not adding a proper description of this issue.

**Comment by vstakhov** [comment]
It looks good but we need a tool to check those keys.

**Comment by left-try** [comment]
Yes. I have already implemented the rapamadm utility for this, but I implemented it in another branch and wanted to open a new pull request for it.

**Comment by vstakhov** [comment]
You can create a stacked PR where one branch will be the base of your another PR. To do that, create a new PR, then press `edit` button and select the appropriate base for your new PR (that should be this branch). Or check docs about `stacked PRs` on Github.

**Comment by left-try** [comment]
Ok. Thank you!

**Pull request merged**